datadog.conf.example

```diff
@@ -47,7 +47,8 @@ api_key:
 # and dice per monitored app (= running Agent Check) on Datadog's backend.
 # create_dd_check_tags: no
 
-# Collect AWS EC2 custom tags as agent tags (requires an IAM role associated with the instance)
+# Collect AWS or OpenStack EC2 custom tags as agent tags
```

Comment:
Does OpenStack need to be mentioned here or can you move it all down to the openstack config options? I think it would make more sense down there.